src/test/java/teammates/test/cases/webapi/GetFeedbackQuestionRecipientsActionTest.java

```diff
@@ -23,12 +52,190 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+
+        ______TS("Invalid intend, should fail");
+        String[] invalidIntendParams = generateParameters(firstSessionInCourse1, 1, Intent.FULL_DETAIL,
```

Comment:
break the line at high level.
```suggestion
        String[] invalidIntendParams = 
                generateParameters(firstSessionInCourse1, 1, Intent.FULL_DETAIL, "", "", "");
```